src/test/java/org/swisspush/reststorage/mocks/DelegatingVertxFileSystem.java

```diff
@@ -171,6 +252,11 @@ public FileSystem deleteRecursive(String s, boolean b, Handler<AsyncResult<Void>
         return delegate.deleteRecursive(s, b, handler);
     }
 
+    @Override
+    public Future<Void> deleteRecursive(String path, boolean recursive) {
+        return null;
```

Comment:
Expected behavior: Delegates call to its *delegate*
Actual behavior: Does nothing.

Reply:
ok